source/_integrations/neato.markdown

```diff
@@ -24,40 +25,38 @@ There is support for the following platform types within Home Assistant:
 - **Switch** - allows you to enable or disable the schedule.
 - [**Vacuum**](#vacuum)
 
-To activate `neato` in your installation, you can set it up from the integration screen or add it to your `configuration.yaml` file.
+## Setup the integration
 
-## Setup the integration via the integrations screen
+1. Visit <https://developers.neatorobotics.com/applications> and create a new app.
 
-Menu: *Configuration* -> *Integrations*
-
-Search for or select **Neato** from the list and configure the integration. You will need to enter your username and password and whether you are using a Neato or Vorwerk device.
-After that, all the entities will automatically show up in Home Assistant.
-
-## Setup the integration via `configuration.yaml`
+<div class='note'>
+You will have to enter an name, a description and your redirect URL.
+If homeassistant runs on <https://hass.example.com> your redirect URL would be <https://hass.example.com/auth/external/callback>.
+Please note that you must use a secure redirect URL (https://).
```

Comment:
Having SSL is not a requirement of the OAuth2 specifications?

Reply:
Unfortunately neato requires that the redirection URL starts with https

![grafik](https://user-images.githubusercontent.com/15657682/101462091-0df65300-393c-11eb-905f-5c6759091000.png)

Reply:
I hate that companies divert slightly and making their own protocols 😢